src/widgets/breadcrumb/breadcrumb.js

```diff
@@ -164,7 +166,7 @@ export default function breadcrumb({
   });
 
   const makeBreadcrumb = connectBreadcrumb(specializedRenderer, () =>
-    unmountComponentAtNode(containerNode)
+    render(null, containerNode)
```

Comment:
have we migrated to this in every component?

Reply:
Yep